### ISSUE TITLE
NTR: CT or MRI: optic nerve mass (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8891]
New term request CT or MRI: optic nerve mass (segal_090730151209): Glioma, tumor included  UMLS: 0346328

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000918
- label: Optic nerve mass
- parents: [Abnormal optic nerve morphology (HP:0000587)](https://hpo.jax.org/app/browse/term/HP:0000587)
- pmid: 
- def: A swelling or enlargement localized to the optic nerve. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: This finding may represent the initial result of CT or MRI studies.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8891
- synonyms:
